### ISSUE TITLE
error message after "5.2 Single cell isoform profile" is finished

### ISSUE THREAD

**Issue by myano-fgin** [gaolabtools/scNanoGPS#27]
Hello Cheng-kai,
I come to the "5.2 Single cell isoform profile".
After the process is finished, there comes error messege.
Is this a problem?

```
Batch LIQA jobs finished !


Time stamp: Tue, 16 Apr 2024 17:51:06

Time elapse: 0 : 23 : 15.58
Traceback (most recent call last):
  File "/home/minoruyano/opt/local/scNanoGPS/reporter_isoform.py", line 201, in <module>
    res_tb[CB].update({options.tx_mapping[line_list[1]] + '_' + line_list[1]: line_list[2]})
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
KeyError: 'ENST00000639447'
```

Best regards,
Minoru

**Comment by shiauck** [comment]
Hi Minoru,

This error is due to no isoform detection in a particular single cell.
When compiling all the single cell isoform results into isoform matrix, it generates the error.
Please check if you got isoform matrix.
Thanks.

Regards,
Cheng-Kai

**Comment by myano-fgin** [comment]
Hi Cheng-Kai,

Thank you reply for my post.
I understand that the error is because of no isoform in a paticular single cell.
Now, this trouble has been solved.

Reviwing the situation when I had got the error message, I found as below.
I have got the error message when I set "--min_gene_no 1" at the previous reporter_expression.py process.
I set this "--min_gene_no" option intentionally in order to get all CB data at this point even though they are with very small number of reads .

According to your advice, I redo the reporter_expression.py process without the option, so that the Minimal number of gene per cell became default "Default: 300".  Then, CBs with very small number were eliminated at the  reporter_expression.py process, then the error message did not come up at this reporter_isoform.py process.

Best regards,
Minoru
